app/components/global/layout/SectionalLayout.js

```diff
@@ -16,12 +17,15 @@ const SectionalLayout = ({ top, side, main }) => (
   <div>
     <TopNavContainer>{top}</TopNavContainer>
     <Flex>
-      <SideNavContainer ml={[0, 0, '8.33%']} width={[0, 0, 3 / 12, 2 / 12]}>
+      <SideNavContainer
+        ml={[0, 0, '8.33%']} // 1 column's worth of spacing (in a 12 column grid)
+        pr={3}
+        pt={18} // To match the top padding on the H1 in the 'main' component
+        width={[0, 3 / 12, 3 / 12, 2 / 12]}
+      >
         {side}
       </SideNavContainer>
-      <MainContainer width={[1, 1, 7 / 12, 7 / 12, 6 / 12]}>
-        {main}
-      </MainContainer>
+      <MainContainer width={[1, 7 / 12, 7 / 12, 8 / 12]}>{main}</MainContainer>
```

Comment:
Is this change of width right? I thought the main content was meant to be full width up until 768px (i.e. above breakpoint 2). https://projects.invisionapp.com/share/GPONO4HMFVX#/screens/326418929

Reply:
Ah!!! My misunderstanding of grid-styled. Thanks :+1: